### DISCUSSION TITLE
island potion effect apply /is command

### DISCUSSION THREAD

**Discussion by Dot0420** [BG-Software-LLC/SuperiorSkyblock2#2017] [Approved Feature Requests]
I would like the island potion effect to be applied when the /is command is entered. Currently, the potion effect is only applied when the /is command is entered in a different world. This is to address that issue.

The island potion is tested by applying it in the following manner: `/is admin seteffect Dot0420 ABSORPTION 1`

**Comment by Dot0420** [comment]
I checked now and it is possible to implement it through API!

**Comment by OmerBenGera** [comment]
It sounds like a bug.
The effect of the island should be applied as you enter the island, no matter from where.
If that's not the case, open an issue on github
